test/index.spec.ts

```diff
@@ -263,18 +264,38 @@ test("createServer: includes cf property on request", async (t) => {
     }`,
   });
   const origin = await listen(t, mf.createServer());
-  const body = JSON.parse((await request(origin))[0]);
-  t.deepEqual(body, {
-    asn: 0,
-    colo: "XXX",
-    country: "XX",
-    httpProtocol: "HTTP/1.1",
-    requestPriority: null,
-    tlsCipher: "",
-    tlsClientAuth: null,
-    tlsVersion: "",
-    timezone: "",
-  });
+  const body: NonNullable<RequestInit["cf"]> = JSON.parse(
+    (await request(origin))[0]
+  );
+
+  t.true(body.asn > 0, "ASN is a valid number");
+  t.is(body.colo.length, 3);
+  t.not(body.city?.length, 0);
+  t.not(body.region?.length, 0);
+  t.is(body.regionCode?.length, 2);
+  t.not(body.metroCode?.length, 0);
+  t.not(body.postalCode?.length, 0);
+  t.is(body.country.length, 2);
+  t.is(body.continent?.length, 2);
+  t.not(body.timezone?.length, 0);
+
+  if (body.latitude) {
+    t.regex(body.latitude, /^-?\d+\.\d+/);
+  } else {
+    t.fail();
+  }
+
+  if (body.longitude) {
+    t.regex(body.longitude, /^-?\d+\.\d+/);
+  } else {
+    t.fail();
+  }
```

Comment:
You're absolutely correct that this is overly verbose. I wanted to simply use the non-null assertion operator (`body.latitude!`), but there is a linting rule that does not permit the non-null assertion operator.

Rather than doing nullish coalescing to an empty string (which will always fail the grep as well), I chose to be explicit without violating the linting rule.

Whatever you're happy with works for me, but I think that disabling the linting rule for that test's function would probably be the most straight-forward.